src/components/nodes/Text.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div class="q-px-md q-pt-md">
+    <q-input
+      filled
+      autogrow
+      v-model="text"
```

Comment:
This `v-model` does not have a corresponding `data` field.

Reply:
Good catch.